openshift/templates/app/pims-api/build-dev.env

```diff
@@ -0,0 +1,7 @@
+APP_NAME=pims
```

Comment:
I have added the .env files for convenience and because most teams include it (see our own frontend openshift templates, they gave them as .param files). 

Also, none of the information there is sensitive, please note that how the SQL server admin password is auto-generated, not set in the .env file.

However, I'm fine taking them out but we will need to add to the documentation instructions on how to create them as per your comment above with `oc process`

Reply:
I lean towards not including them, just documentation on what to do.  Otherwise we risk including sensitive information (by mistake) and it creates duplication of effort when ensuring the default values are correct in the YAML.

Reply:
Agreed. I will update the PR

Reply:
Just a reminder to update the readme as well after you remove the env files.

Reply:
yeah, thanks. working on it